### PR TITLE
fix broken jira link

### PR DESCRIPTION
**Pull request by evernat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ To use this plugin visit the `Manage Jenkins` -> `Disk usage` page.
 
 # Resources
 * Continuous Integration: [![Build Status](https://jenkins.ci.cloudbees.com/buildStatus/icon?job=plugins/cloudbees-disk-usage-simple-plugin)](https://jenkins.ci.cloudbees.com/job/plugins/job/cloudbees-disk-usage-simple-plugin)
-* Issues Tracking: [Jira](https://issues.jenkins-ci.org/browse/JENKINS/component/20652)
+* Issues Tracking: [Jira](https://issues.jenkins-ci.org/browse/JENKINS-45943?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20cloudbees-disk-usage-simple-plugin)
```
Comment (darxriggs) [inline README.md]: I suggest to use `https://issues.jenkins-ci.org/issues/?jql=project+%3D+JENKINS+AND+component+%3D+cloudbees-disk-usage-simple-plugin`. This does not link a specific issue.

**Comment by darxriggs** [comment]
@ndeloof can you have a look at this?

File: README.md (hunk 1 above)
Comment (darxriggs) [reply README.md]: The suggested URL is also included in pull request #24.

**Comment by aheritier** [comment]
Fixed by #24

**Pull request closed without merging**